visualization/utils/common.py

```diff
@@ -0,0 +1,386 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+import functools
+import json
+import logging
+import re
+import subprocess
+from collections import defaultdict
+from typing import Callable, Dict, Iterable, List, Optional, Set, Tuple, Union
+
+from flask import current_app
+from joblib import Parallel, delayed
+from settings.constants import (
+    ANSWER_FIELD,
+    ERROR_MESSAGE_TEMPLATE,
+    OUTPUT,
+    PARAMETERS_FILE_NAME,
+    QUESTION_FIELD,
+    RESULTS_PATH,
+    STATS_KEYS,
+)
+
+from nemo_skills.inference.prompt.few_shot_examples.examples_gsm8k import examples_map as examples_gsm8k

```

Comment:
Can you just import all examples like done here https://github.com/Kipok/NeMo-Skills/blob/main/nemo_skills/inference/prompt/utils.py#L27